packages/pg-v5/test/commands/copy.js

```diff
@@ -139,7 +141,7 @@ describe('pg:copy', () => {
     it('copies', () => {
       return cmd.run({ app: 'myapp', args: { source: 'HEROKU_POSTGRESQL_RED_URL', target: 'myotherapp::DATABASE_URL' }, flags: { confirm: 'myapp' } })
         .then(() => expect(cli.stdout, 'to equal', ''))
-        .then(() => expect(cli.stderr, 'to equal', `Starting copy of RED to BLUE... done\n${credentialWarningText()}${copyingText()}`))
+        .then(() => expect(unwrap(cli.stderr), 'to equal', `Starting copy of RED to BLUE... done ${credentialWarningText()}${copyingText()}`))
```

Comment:
What's the issue to be aware of on Windows here, to make sure we don't break this in the future? I sort of understand what this is doing, but not why.

Reply:
Windows used a different error character causing this to fail.

Reply:
`unwrap()` strips the newlines as well as the leading arrow character which is just a bang on windows. It essentially normalizes the error message so it's consistent to test against and won't change based on the terminal width or OS.

It was a bad idea though and I improved this in oclif by having tests catch the exception itself (before it gets stylized and indented and everything). Though this problem will still exist for warnings.